powermon/device.py

```diff
@@ -58,7 +58,7 @@ def add_command(self, command: Command) -> None:
         if command is None:
             return
         # get command definition from protocol
-        command.set_command_definition(self.port.protocol.get_command_definition(command.code))
+        command.set_command_definition(self.port.protocol.get_command_with_command_string(command.code))
```

Comment:
`get_command_from_string` or `definition_from_string` (to align more with from_config, to_dto etc)